cli/ceph/orch/tuned_profile.py

```diff
@@ -17,7 +17,9 @@ def apply(self, spec_file):
         """
         cmd = f"{self.base_cmd} apply -i {spec_file}"
         out = self.execute(sudo=True, cmd=cmd)
-        if isinstance(out, tuple):
+        if out[0] == "":
```

Comment:
Why we need this change ? Can't it be handled in test case workflow ?

Reply:
Hi @vamahaja 
No it can't handle from test case workflow since as of now for negative test case (Command failed with error) out[0] is black and error is updating in out[1]

Reply:
then use try..catch in the test workflow

Reply:
Updated